python/test/t_DistributionFactory_std.py

```diff
@@ -46,9 +47,77 @@
             dist = factory.build(sample)
         except Exception:
             ok = True
-        print(weird + ":", "ok" if ok else "fail")
-        assert ok, str(factory) + " vs " + weird
+            print(weird + ":", "ok" if ok else "fail")
+            assert ok, str(factory) + " vs " + weird
 
 factory = ot.DistributionFactory.GetByName("PoissonFactory")
 print(factory)
 assert factory.getImplementation().__class__.__name__ == "PoissonFactory", "wrong name"
+
+# Now, test each factory for the partial estimation of any subset of its parameters
+
+for factory in factories:
+    print("factory=", factory)
+    try:
+        factoryName = factory.getImplementation().getClassName()
+        aEps = 0.01e100
+        rEps = 0.01e100
+        if factoryName == "BetaFactory":
+            aEps = 0.05
+            rEps = 0.05
+        if factoryName == "FisherSnedecorFactory":
+            aEps = 0.25
+            rEps = 0.25
+        if factoryName == "LogisticFactory":
+            aEps = 0.025
+            rEps = 0.025
+        if factoryName == "LogNormalFactory":
+            aEps = 0.02
+            rEps = 0.02
+        if factoryName == "ParetoFactory":
+            aEps = 0.15
+            rEps = 0.15
+        if factoryName == "RiceFactory":
+            aEps = 0.3
+            rEps = 0.3
+        if factoryName == "StudentFactory":
+            aEps = 0.15
+            rEps = 0.15
+        if factoryName == "StudentFactory":
+            aEps = 0.15
+            rEps = 0.15
+        if factoryName == "TrapezoidalFactory":
+            aEps = 1.0
+            rEps = 1.0
+        if factoryName == "TriangularFactory":
+            aEps = 0.025
+            rEps = 0.025
+        if factoryName == "VonMisesFactory":
+            aEps = 0.02
+            rEps = 0.02
+        if factoryName == "NegativeBinomialFactory":
+            aEps = 0.05
+            rEps = 0.05
```

Comment:
it would be simpler to write this as a dictionnary, you dont even need to store both values since they're equal